packages/generate-pdf/package.json

```diff
@@ -25,7 +25,7 @@
     "dist"
   ],
   "engines": {
-    "node": ">=12",
+    "node": ">=14",
```

Comment:
Node v12 caused issues when importing/executing the script built by parcel v2, so we remove support for it.